src/core/pure-js/deck-js.js

```diff
@@ -240,7 +240,8 @@ export default class DeckGLJS {
       // Helps debug layer picking, especially in framebuffer powered layers
       drawPickingColors: this.props.drawPickingColors
     });
-    this.props.onAfterRender({gl}); // TODO - should be called by AnimationLoop
+    // TODO - should be called by AnimationLoop
+    this.props.onAfterRender({gl, canvas: this.props.canvas});
```

Comment:
You can use [`gl.canvas`](https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/canvas)

Reply:
Thanks for the suggestion, fixed